server/app/controllers/service-controller.js

```diff
@@ -0,0 +1,119 @@
+const serviceService = require('../services/service-service');
+const { Service } = require('../models/service');
+const { checkStructure, createUserError } = require('../../utils');
+
+/**
+ * Fetch all the services from the database.
+ *
+ * @param req Request
+ * @param res Response
+ * @return {Promise.<void>} Nothing
+ */
+async function getAllServices(req, res) {
+    const services = await serviceService.getAllServices();
+
+    res.json(services);
+}
+
+/**
+ * Create a service.
+ *
+ * @param req Request
+ * @param res Response
+ * @return {Promise.<void>} Nothing
+ */
+async function createService(req, res) {
```

Comment:
Actually, we decided to create an array of services, see swagger for more details.

Reply:
We will do this in another PR